rpcperms/interceptor.go

```diff
@@ -948,21 +945,22 @@ func (r *InterceptorChain) acceptRequest(requestID uint64,
 	return nil
 }
 
-// interceptResponse sends out an intercept request for an RPC response. Since
+// interceptMessage sends out an intercept request for an RPC response. Since
 // middleware that hasn't registered for the read-only mode has the option to
-// overwrite/replace the response, this needs to be handled differently than the
-// request/auth path above.
-func (r *InterceptorChain) interceptResponse(ctx context.Context,
-	requestID uint64, isStream bool, fullMethod string,
-	m interface{}) (interface{}, error) {
```

Comment:
Can also go `interface{}` -> `any`, here. Though maybe we should just do a single PR that does a search and replace everywhere.

Reply:
We haven't actually updated Go to 1.18 in `go.mod`. So I think we shouldn't use any 1.18 features just yet.